### PR TITLE
Add volunteer signup

### PR DESCRIPTION
**Pull request by CedricLSM**
Set up addVolunteerDatain Database.js and finished handleSubmit function in SignUpVolunteer to add additional volunteer details into the database

### CODE REVIEW

File: src/util/Database.js [hunk 1]
```diff
@@ -16,9 +16,15 @@ export default class Database {
...
-  }
+  static addVolunteerData(volunteerUserId, additionalData) {
+    // const querySnapshot = await firebase
+    firebase
+      .firestore()
+      .collection("users")
+      .doc(volunteerUserId)
+      .update(additionalData)
+      .catch(error => console.error("Error adding user details to DB: ", error));
+    }
```
Comment (wennyyustalim) [inline src/util/Database.js:27]: nit: the indentation doesn't look right

File: src/util/Database.js [hunk 2]
```diff
@@ -16,9 +16,15 @@ export default class Database {
     // TODO
   }
 
-  static addVolunteerData(userId, data) {
-    // TODO
-  }
+  static addVolunteerData(volunteerUserId, additionalData) {
+    // const querySnapshot = await firebase
```
Comment (wennyyustalim) [inline src/util/Database.js:20]: let's remove commented code :) why do we lose the `await`?

File: src/util/Database.js (hunk 1 above)
Comment (CedricLSM) [reply src/util/Database.js:27]: Adjusted it! Thank you for the catch.

File: src/pages/SignUpVolunteer.js [hunk 3]
```diff
@@ -40,13 +46,21 @@ function SignUpResearcher() {
...
 
-  const handleSubmit = values => {
-    //TODO: authenticate to allow login
-
-    // Db.addVolunteerData(useContext(UserContext)?.uid, ...); TODO
-    // navigate to next page
-
+  const handleSubmit = async values => {
+    try {
+      await Db.addVolunteerData(userId, values).then;
```
Comment (seeziyang) [inline src/pages/SignUpVolunteer.js:51]: Similar issue as the previous PR, committed a fix for it and will explain it in that!

**Pull request merged**